fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/LeadingJoin.java

```diff
@@ -41,7 +41,7 @@ public Plan rewriteRoot(Plan plan, JobContext jobContext) {
                     (LeadingHint) leadingHint, ((LeadingHint) leadingHint)
                         .getLeadingTableBitmap(jobContext.getCascadesContext().getTables())));
             if (leadingHint.isSuccess()) {
-                jobContext.getCascadesContext().getConnectContext().getSessionVariable().setDisableJoinReorder(true);
+                jobContext.getCascadesContext().getConnectContext().getStatementContext().setDisableJoinReorder(true);
```

Comment:
u could refer to `org.apache.doris.qe.SessionVariable#enableFallbackToOriginalPlannerOnce` to change session variable for only one sql